### ISSUE TITLE
(BUG) Solar Panels won't connect to power cables

### ISSUE THREAD

**Issue by Vaygrim** [TechReborn/TechReborn#1560]
Minecraft 1.12.2
Forge 2705
RebornCore 1.12.2-3.9.0.315
TechReborn 1.12.2-2.16.0.746-universal

Latest update to things (I *think* it's TechReborn, but I know we updated RebornCore as well), our Solar Panel blocks (all grades tested) no longer connect to adjacent power cables on ANY of their sides. Yes we even checked the tops where the solar bit is.

This is not just a 'visual glitch', the solar panels now just accrue power and never output them. NOTE: Putting a battery block directly underneath them DOES work (tried TechReborn, Thermal Expansion, all fine). The blocks just seem to refuse to cooperate with power cables specifically.

Help?

**Comment by modmuss50** [comment]
Seems like an odd one, someone else has noted cables not connecting to machines. I am busy for a few hours but will be able to take a look later and get a hotfix out if needed. 

Thanks for the detailed and clear issue.

**Comment by Vaygrim** [comment]
I can try to get screenshots if needed but since it's not causing an actual _crash_ .. there's little information i can pass on beyond what I already have.

I will note, we checked with TR Copper, Tin and Gold cables as well as Glass Fibre cables. None of them would connect and draw power. Tried the bottom three tiers of Thermal Dynamics cables (Leadstone, Hardened, and Reinforced?), same result.

Stuck a Thermal Dynamics Leadstone Energy Cell under one, and BINGO it started transferring power immediately. Not a clue beyond that. :(
